src/utils/user_utils.ts

```diff
@@ -41,41 +41,51 @@ export function displayUsername(
     return name;
 }
 
-export function rolesIncludePermission(roles: string, permission: string): boolean {
-    const rolesArray = roles.split(' ');
-    return rolesArray.includes(permission);
+export function spaceSeparatedStringIncludes(spaceSeparated: string, item: string): boolean {
```

Comment:
Unrelated: renamed this func.

Reply:
I think you can actually just replace this entire function with `string.includes(item)` no need for the split and no need for this function in general

Reply:
I guess this is more accurate if you have a permission that is a substring of another permission for example `sysconsole_usermanagement` and `read_sysconsole_usermanagement` ... you could get a false positive, ignore my above comment